packages/docs/source/components/tag.md

```diff
@@ -0,0 +1,23 @@
+# Tag
+
+Tags are used to help describe and differentiate an entity or object. Think of them as “adjectives” in your UI toolbox that help navigating and parsing content easier.
+
+## Usage
+
+As stated above, Tags are used to describe an entity or object, not represent the object itself. One or many Tags may be applied to an entity or object.
```

Comment:
This sentence can be re-worded a bit. I think we should be more explicit around what is an entity and what is an object.

Reply:
(even with the example that follows it - this might just be me though. Will defer to @joejezowski-okta)

Reply:
Makes sense - I think @joejezowski-okta and I were trying to cast a broad language net. We could def remove one or the other, though.